library_generation/test/resources/integration/google-cloud-java/pr-description-golden.txt

```diff
@@ -1,23 +1,19 @@
 This pull request is generated with proto changes between googleapis commit a17d4caf184b050d50cacf2b0d579ce72c31ce74 (exclusive) and 4ce0ff67a3d4509be641cbe47a35844ddc1268fc (inclusive).
```

Comment:
Can we make these two commit a link too?

Reply:
Done.